src/System.Threading.Tasks/tests/Task/TaskStatusTest.cs

```diff
@@ -159,7 +159,7 @@ internal void RealRun()
                         // without Detached options and current status of the child isn't RanToCompletion or Faulted yet
                         //
 
-                        Task.Delay(100).Wait();
+                        Task.Delay(1).Wait();
```

Comment:
Is there a material difference between using this and doing the `SpinWait` stuff above? I get that you were just lowering the duration of the existing call here; just asking for clarification.